composer.json

```diff
@@ -11,10 +11,11 @@
     ],
     "require" : {
         "php": ">=5.4",
+        "swiftmailer/swiftmailer": "^5.1",
         "tijsverkoyen/css-to-inline-styles": "^1.5"
     },
     "require-dev" : {
-        "swiftmailer/swiftmailer": "^5.1"
+        "phpunit/phpunit": "^4.0"
```

Comment:
@nclsHart Do you think we could allow both 4.x and 5.x versions of PHPUnit here? Or just upgrade to the 5.x latest stable version?